alpha/opentracev2.h

```diff
@@ -50,7 +50,7 @@ struct OT_ConnectionRecord
 //
```

Comment:
don't commit files with no change

Reply:
Sorry I auto-formatted all the files, and committed all 👎 thumbsdown for me